lib/chai/core/assertions.js

```diff
@@ -2145,4 +2145,27 @@ module.exports = function (chai, _) {
       , 'expected #{this} to not be frozen'
     );
   });
+
+  /**
+   * ### .finite
+   *
+   * Asserts that the target is a finite number. Unlike `.a('number')`, this will fail for `NaN` and `Infinity`
```

Comment:
Also, I know I'm being nitpicky, but maybe we could have a final dot in this phrase.